### PR TITLE
Feat/146 date of next new moon

### PR DESCRIPTION
**Pull request by Logan-Campbell**
Functionality for getting next date of new moon and test cases

### CODE REVIEW

**Review by mrstarch — APPROVED** [review]

File: tests/test_DateOfNextNewMoon.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from control import getDateOfNextNewMoon
```
Comment (FreddieBanfield) [inline tests/test_DateOfNextNewMoon.py]: Please add these tests (call the tests you created in test_DateOfNectNewMoon.py) to the test_all.py file, as we will run all unit tests at one time, not as individual files.
image

File: AstroCal/control/control.py [hunk 2]
```diff
@@ -206,3 +206,64 @@ def getMoonStatus():
 
     moon_percent_rounded = round(moon_percent)
     return result, "Illumination " + str(moon_percent_rounded) + "%"
+    
+    
+
```
Comment (FreddieBanfield) [inline AstroCal/control/control.py:209]: Not seeing any changes to console_output.py. That means there is no way for me to see this in the console at the moment. Please add an extra option in the moon_menu in console_output.py. 
image

**Review by FreddieBanfield — CHANGES_REQUESTED** [review]
Please may you complete these two changes.

**Review by FreddieBanfield — APPROVED** [review]
Looks great!

**Pull request merged**